p/scripts/main.js

```diff
@@ -962,7 +962,23 @@ function init_column_categories() {
 	};
 }
 
+function init_shortcuts_helper() {
+	document.addEventListener("keydown", function (event) {
+		if (event.shiftKey && event.key === "?") {
+			const activeElement = document.activeElement;
+			const isInputField = activeElement.tagName === "INPUT" ||
```

Comment:
It may be easier to read along these lines, but then again it may be faster as written.

More important note the check that activeElement isn't null and including `SELECT` with the list of elements.

The activeElement thing may be outdated but it wouldn't hurt either, cf. `https://github.com/ten1seven/what-input/issues/38` for example.
```js
const isInputField = activeElement && (
                     ["INPUT", "TEXTAREA", "SELECT"].includes(activeElement.tagName) ||
                     activeElement.isContentEditable
                     );
```